rasa/core/agent.py

```diff
@@ -89,23 +89,23 @@ def _load_and_set_updated_model(
             agent.interpreter if agent.interpreter is not None else RegexInterpreter()
         )
 
-    domain = None
-    if core_path:
-        domain_path = os.path.join(os.path.abspath(core_path), DEFAULT_DOMAIN_PATH)
-        domain = Domain.load(domain_path)
-
     try:
```

Comment:
Shouldn't we wrap everything in a `try ... except`? NLU model loading might go wrong just as likely, right?